tests/test_invessel_build.py

```diff
@@ -104,14 +104,19 @@ def test_ivb_basics(invessel_build):
 def test_ivb_construction(invessel_build):
 
     num_components_exp = 2
+    len_loci_pt_exp = 3
 
     remove_files()
 
     invessel_build.populate_surfaces()
     invessel_build.calculate_loci()
     invessel_build.generate_components()
 
+    rib_loci = invessel_build.Surfaces['component'].get_loci()[0]
+    
+
     assert len(invessel_build.Components) == num_components_exp
+    assert len(rib_loci[0]) == len_loci_pt_exp
```

Comment:
Why do we expect only 3 pts? I'm not sure I understand the shape here?

Is this basically `len(poloidal_angles_exp) - 1`

Reply:
I expect an element of `rib_loci` to have an iterable of (x,y,z).

I think perhaps I am too focused on testing for the issue I set out to address in the first place, maybe a better testing method would be to have expected values for x,y,z for a specific point on a specific rib and check those?

For example call `ivb.get_loci()` or `Surface.get_loci()`, then get the loci for a single rib, and from that a single loci to check? This would not only check that the `get_loci()` methods are correct, but that the points being calculated are also not changing. I think there might be floating point error to worry about with that sort of comparison though?

Reply:
I think we could at least test for the number of points on the rib? That would probably be enough

Reply:
that's getting tested in line 98 already, and didn't catch that loci weren't being returned since the length was as expected, it was just filled with methods rather than loci.